src/CMakeLists.txt

```
@@ -126,8 +126,8 @@ target_sources(shared
 
 target_compile_definitions(shared
 	PUBLIC
-		"MUMBLE_VERSION=${version}"
-		"MUMBLE_VERSION_STRING=${version_short}"
+		"MUMBLE_VERSION=${PROJECT_VERSION}"
+		$<$<NOT:$<STREQUAL:"${version}","">>:MUMBLE_VERSION_STRING=${version}>
```

I think while we're on it, `MUMBLE_RELEASE_ID` would be more fitting for this define since
1. We already have `MUMBLE_VERSION` so another definition `MUMBLE_VERSION_STRING` should be expected to be the same thing, but stringified. This is not the case though
2. This doesn't necessarily have anything to do with the actual version. It's just a String identifying the version. Thus I think `ID` is the better suffix here.

Plus I'd vote for adding an explanatory comment like so:
```suggestion
		# Only define this macro, if the version variable is actually set
		$<$<NOT:$<STREQUAL:"${version}","">>:MUMBLE_RELEASE_ID=${version}>
```